src/features/location/components/SearchLocationModal.tsx

```diff
@@ -120,6 +122,10 @@ export const SearchLocationModal = ({
     onModalHideRef.current = showVenueModal
   }
 
+  const AddTracker = useCallback(() => {
```

Comment:
```suggestion
  const addTracker = useCallback(() => {
```

c'est bizarre ta majuscule là :D